src/Projection/User/UserProjector.php

```diff
@@ -105,4 +106,28 @@ public function onTodoWasMarkedAsDone(TodoWasMarkedAsDone $event)
 
         $stmt->execute();
     }
+
+    /**
+     * @param TodoWasReopened $event
+     * @throws \RuntimeException if data of the the assigned user can not be found
+     */
+    public function onTodoWasReopened(TodoWasReopened $event)
+    {
+        $user = $this->userFinder->findUserOfTodo($event->todoId()->toString());
+
+        if (! $user) {
+            throw new \RuntimeException(
```

Comment:
use this exception: https://github.com/prooph/proophessor-do/blob/master/src/Model/User/Exception/UserNotFound.php